modules/shimmer/1.0/envs/shimmer.env.yaml

```diff
@@ -0,0 +1,142 @@
+name: null
```

Comment:
This yaml file should be in `lcr-modules/shimmer/shimmer.yaml` and there should be a symlink from `modules/shimmer/1.0/envs` pointing to the actual yaml file.